lib/ansible/modules/unarchive.py

```diff
@@ -980,6 +980,10 @@ def main():
         else:
             res_args['changed'] = True
 
+    top_folder_name = handler.files_in_archive[0].split('/')[0]
+    file_args['path'] = "{0}/{1}".format(dest, top_folder_name)
```

Comment:
use `%s` notation as this needs to be python 2.6/2.7 compatible

Reply:
I have updated the PR to use `%s` notation as requested